### DISCUSSION TITLE
Binding FRSKY S6R

### DISCUSSION THREAD

**Discussion by mikeyJRB** [EdgeTX/edgetx#1822] [Q&A] [answered]
All, I updated my RM TX16S to 2.7 RC2 and my MULTI Protocol firmware to V1.3.3.7 AETR this weekend from 2.6 and V1.1.? AETR and after doing so I could not connect to my model using a FRSKY S6R receiver.  I tried to bind the receiver several times unsuccessfully.   I was able to bind to a Radiomaster R168 receiver which uses the same RF Protocol successfully, so I am wondering if the S6R receiver firmware needs to be updated.  Any suggestions you have as to what to do next is appreciated and I thank you in advance.

**Comment by pfeerick** [comment, marked as answer]
Firstly, let me preface this with "I have never used a S6R, but do use other Frsky gear" 😉.  Since you have the MPM, you *should* be able to try both the Frsky (D16) and Frsky X2 (D16) protocols, as if it's just the usual frsky ACCST version nonsense, the MPM can speak both versions without requiring the receiver to be flashed. Judging from what John says about binding it, it looks like it *should*  be the first generation Frsky D16... https://youtu.be/BObseyO1cjE?t=155

**Reply by mikeyJRB** [reply]
Peter, thank you for responding.  Indeed, the Frsky X2 (D16) RF protocol allowed me to bind the receiver and everything seems to be working.   Before flying though I will run through the stabilization parameters to make sure everything is set properly.   I wonder still if flashing the receiver with the latest firmware would be beneficial.   Some YouTube videos show this can be done using the RM TX16S transmitter by loading the receiver firmware onto the TX16S SD card and performing an external flashing using a modified servo lead from the external bay UART to the S.PORT of the receiver.

**Reply by pfeerick** [reply]
Fantastic! :) Given it bound with X2, this suggests it probably is probably on the a recent firmware already since X2 is for the second generation ACCST RF link... there are two updates available - one being the initial "ACCST D16 v2.1.0" support, and an update that seems to be specifically for compatibility with the X20 transmitter (for FCC regions). It certainly shouldn't hurt if you want to update it still, but probably isn't necessary, and the videos on updating frsky receivers via module bay by Lee (painless360) should be a good guide on doing the update.

**Comment by MadMonkey87** [comment]
I had a similar problem, then I found the reason and solution in my case. Mine was an EU receiver so I switched the protocol to FrSkyX2 - LBT and it worked!

**Reply by pfeerick** [reply]
Yup, that sounds like the normal FrSky dance... you have ACCST V1 and V2, and then FCC and EU-LBT versions of each... 🤦At least with the MPM you can just keep changing options until you find one that works... prior to that... you had flash the receiver with the right firmware (as you didn't want to do your transmitter as it would then break for all your other models). But sometimes you may have to do that depending on receiver firmware support... what fun times! 

Glad you got it working, hopefully that will help someone else if they get stuck! 😃
